mkdocs.yml

```diff
@@ -46,3 +53,58 @@ extra:
     - icon: fontawesome/solid/paper-plane
       link: mailto:info@teskalabs.com
       name: info@teskalabs.com
+
+nav:
+  - About:
+    - index.md
+    - reference/features.md
+
+
+  - Administration guide:
+    - admin/index.md
+    - Installation:
+      - installation/quick-start.md
+      - installation/provisioning.md
+
+    - Configuration:
+      - config/basic-configuration.md
+      - Credentials:
+        - config/credentials/index.md
+        - config/credentials/credentials-policy.md
+        - config/credentials/providers.md
+      - Authentication and login:
+        - config/external-login.md
+      - config/notifications-configuration.md
+  
+    - Integrating apps:
+      - integrations/index.md
+      - integrations/oauth2-introspect.md
+      - Connecting other apps:
+        - integrations/cookies.md
+        - integrations/elk.md     
+        - integrations/the-hive.md
+  
+    - Development:
+      - REST API:
+        - development/rest-api.md
+        - development/postman.md
+      - development/oauth2.md
+  
+    - Reference:
+      - reference/architecture.md
+      - reference/components.md
+      - reference/resources.md
+      - reference/clients.md
+      - reference/session.md
+
+
+  - User guide (UI):
+      - user/index.md
+      - user/key-terms.md
+      - user/login.md
+      - user/credentials.md
+      - user/tenants.md
+      - user/resources.md      
+      - user/roles.md
+      - user/clients.md
+      - user/sessions.md
```

Comment:
in my view, the User guide is meant for common users who use seacat auth only for logging in and possibly updating their own account settings (email address, passwords etc.). they don't need to know anything about credentials, resources or tenants - all of that is admin stuff.

i think we should only have two pages here - one about login and the other about account settings.

the rest should probably go to the configuration section (for now at least)